Add files via upload

ZavialovaAV opened the pull request.

ZavialovaAV committed:
Add files via upload

ZavialovaAV committed:
Add files via upload

вариант 6

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+else {
+cout << "Точки не находятся на одной прямой." << endl;
+double angle = angleBetweenSegments(A, B, C);
+angle = angle * 180.0 / 3.14; // Перевод из радиан в градусы
```
M_PI вместо 3.14

ZavialovaAV committed:
Add files via upload

вариант 6

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+* @param p2 Вторая точка.
+* @return double Расстояние между точками.
+*/
+double distance(Point p1, Point p2);
```
double distance(const Point p1, const Point p2);

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+* @return true Если точки лежат на одной прямой.
+* @return false Если точки не лежат на одной прямой.
+*/
+bool arePointsCollinear(Point A, Point B, Point C);
```
bool arePointsCollinear(const Point A, const Point B, const Point C);

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+
+double angleBetweenSegments(Point A, Point B, Point C) 
+{
+double AB = distance(A, B);
```
const double AB = distance(A, B);

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+double angleBetweenSegments(Point A, Point B, Point C) 
+{
+double AB = distance(A, B);
+double BC = distance(B, C);
```
const double BC = distance(B, C);

BalakinaEP commented on задание 3.cpp:
```diff
@@ -0,0 +1,117 @@
+{
+double AB = distance(A, B);
+double BC = distance(B, C);
+double AC = distance(A, C);
```
const double AC = distance(A, C);

ZavialovaAV committed:
Add files via upload

Даны 3 точки A(x1, y1), B(x2, y2), C(x3, y3). Определить, будут ли они расположены на одной прямой. Если нет, то вычислить угол В.

BalakinaEP commented on задание 3 (2).cpp:
```diff
@@ -0,0 +1,118 @@
+
+bool arePointsCollinear(Point A, Point B, Point C) 
+{
+return ((B.y - A.y) * (C.x - B.x) == (C.y - B.y) * (B.x - A.x));
```
не убедили
